src/Plugin/Field/FieldType/SectionsItem.php

```diff
@@ -129,6 +130,16 @@ public function setValue($values, $notify = TRUE) {
     if (isset($this->properties['json'])) {
       unset($this->properties['json']);
     }
+    if ($values['json']) {
```

Comment:
```suggestion
    if (!empty($values['json'])) {
```
Ensure the json key exists?

Reply:
Thanks @jamiehollern for catching this! Fixed!